lib/yggdrasil/player.ex

```diff
@@ -12,9 +13,8 @@ defmodule Yggdrasil.Player do
     Supervisor.add_player user_id, [owner_pid, push_msg]
   end
 
-  def run_cmd(user_id, cmd = %Message{type: :command}) do
-    player_pid = Registry.get_player user_id
-    GenServer.cast(player_pid, {:run_cmd, cmd})
+  def run_cmd(user_id, cmd) do
+    Command.execute cmd, user_id
```

Comment:
Yeah I agree with the crashing however one thing I'm curious about and noted above do we want the command to know about the player? 

This is the player module, it know's how to talk to players, so I kind of would rather see

```elixir
def run_command(user_id, cmd) do
  notify user_id, Command.execute(cmd)
end
```
what concerns me is the side effects in Command.execute. Like for now it's simple but just kind of worried as this grows into thinks like attack commands and other action commands what does that look like.

I realize it may be the point of side effects, but just a concern.